platform/ios/docs/guides/Info.plist Keys.md

```diff
@@ -22,12 +22,29 @@ If you have implemented custom opt-out of Mapbox Telemetry within the user inter
 
 ## MGLIdeographicFontFamilyName
 
-The name of the font family to use for client-side text rendering of CJK ideographs. Set this to the name of a font family which will be available at run time, e.g. `PingFang TC` (iOS 9+), `Heiti TC` (iOS 8+), another appropriate built-in font, or a font provided by your application. Note that if a non-existent font is specified, iOS will fall back to using Helvetica which is likely not to include support for the glyphs needed to render maps in your application.
+The name of the font family to use for client-side text rendering of CJK ideographs.
+
+Currently only used for CJK glyphs. Changing this at run time is not currently
+supported. By default, client-side rendering is enabled for CJK glyphs using Apple
+system default font.
+
+Set `MGLIdeographicFontFamilyName` in your containing app's Info.plist to a string
```

Comment:
Is it possible to change this to:

Set `MGLIdeographicFontFamilyName` in your containing app's Info.plist to the following values:
A string value for using your specific local font which will be available at run time,
e.g. "PingFang TC", another appropriate built-in font, or a font provided by your application.
Note that if a non-existent font is specified, iOS will fall back to using Helvetica which is likely not to include support for the glyphs needed to render maps in your application.

An array of font family names. Once the front ones are unavailable, it will fallback to others until
using default system font.

A boolean value `NO` for using your custom remote font.

Reply:
Let’s break the different possible plist values onto new lines:

> Set `MGLIdeographicFontFamilyName` in your containing application's Info.plist to font family name(s) that will be available at run time, such as “PingFang TC” or “Marker Felt”. This plist key accepts:
>
> - A string value of a single font family name.
>
> - An array of font family names. Fonts will be used in the defined order, eventually falling back to default system font if none are available.
>
> - A boolean value `NO` to disable client-side rendering of CJK glyphs — remote fonts specified in your style will be used instead.

Please also update this text in `MGLRendererConfiguration.h`.

Reply:
I also have a question about this line:

> Note that if a non-existent font is specified, iOS will fall back to using Helvetica which is likely not to include support for the glyphs needed to render maps in your application.

Is this accurate? Are we sure it’s Helvetica (and not SF, the system font)? Are we saying the system font doesn’t fully support CJK glyphs?